mephisto/client/review_app/server/api/views/units_approve_view.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Optional
+
+from flask import current_app as app
+from flask import request
+from flask.views import MethodView
+from werkzeug.exceptions import BadRequest
+
+from mephisto.data_model.unit import Unit
+from mephisto.data_model.worker import Worker
+
+
+class UnitsApproveView(MethodView):
+    def post(self) -> dict:
+        """Approve worker's input"""
+
+        data: dict = request.json
+        unit_ids: Optional[str] = data.get("unit_ids") if data else None
+        feedback = data.get("feedback") if data else None  # Optional
```

Comment:
OK I think there's confusion here that could be improved on the documentation/nomenclature side:
`feedback` elsewhere in `AgentState` refers to a _worker_'s feedback about a task to the requester. We don't use the `feedback` field very frequently though, but use here and changes elsewhere do introduce a bit of chance for confusion.